nixos/modules/services/x11/desktop-managers/pantheon.nix

```diff
@@ -81,9 +81,7 @@ in
 
     services.xserver.displayManager.lightdm.greeters.pantheon.enable = mkDefault true;
 
-    # If not set manually Pantheon session cannot be started
-    # Known issue of https://github.com/NixOS/nixpkgs/pull/43992
-    services.xserver.desktopManager.default = mkForce "pantheon";
+    services.xserver.displayManager.defaultSession = "pantheon";
```

Comment:
I'm not sure if this is needed anymore, or if it has any effects on https://github.com/elementary/greeter/issues/368

Reply:
Hmm, I removed it in one of the reverted commits.